app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -155,122 +155,136 @@ class AppVersionActivityTest {
-      onView(
-        withText(
-          String.format(
-            context.resources.getString(R.string.app_last_update_date),
-            lastUpdateDate
-          )
-        )
-      ).check(
-        matches(isDisplayed())
-      )
-    }
-  }
-
-  @Test
-  fun testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly() {
-    launchAppVersionActivityIntent().use { scenario ->
-      onView(isRoot()).perform(orientationLandscape())
-      val lastUpdateDate = scenario.convertTimeStampToDate(context.getLastUpdateTime())
-      onView(
-        withId(
-          R.id.app_version_text_view
-        )
-      ).check(
-        matches(
+    ActivityScenario.launch(AppVersionActivity::class.java).use { scenario ->
+      scenario.onActivity { activity ->
+        val lastUpdateDate = scenario.convertTimeStampToDate(context.getLastUpdateTime())
+        onView(
           withText(
             String.format(
               context.resources.getString(R.string.app_version_name),
               context.getVersionName()
             )
           )
-        )
-      )
-      onView(
-        withId(
-          R.id.app_last_update_date_text_view
-        )
-      ).check(
-        matches(
+        ).check(matches(isDisplayed()))
+        onView(
           withText(
             String.format(
               context.resources.getString(R.string.app_last_update_date),
               lastUpdateDate
             )
           )
-        )
-      )
+        ).check(matches(isDisplayed()))
+      }
     }
   }
 
+  @Test
+  fun testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly() {
+    ActivityScenario.launch(AppVersionActivity::class.java).use { scenario ->
+      scenario.onActivity { activity ->
+        onView(isRoot()).perform(orientationLandscape())
+        val lastUpdateDate = scenario.convertTimeStampToDate(context.getLastUpdateTime())
+        onView(
+          withId(R.id.app_version_text_view)
+        ).check(
+          matches(
+            withText(
+              String.format(
+                context.resources.getString(R.string.app_version_name),
+                context.getVersionName()
+              )
+            )
+          )
+        )
+        onView(
+          withId(R.id.app_last_update_date_text_view)
+        ).check(
+          matches(
+            withText(
+              String.format(
+                context.resources.getString(R.string.app_last_update_date),
+                lastUpdateDate
+              )
+            )
+          )
+        )
+      }
+    }
+  }
   @Test
   fun testAppVersionActivity_loadFragment_onBackPressed_displaysAdministratorControlsActivity() {
+
     ActivityScenario.launch<AdministratorControlsActivity>(
-      launchAdministratorControlsActivityIntent(
-        internalProfileId = 0
-      )
-    ).use {
+      launchAdministratorControlsActivityIntent(internalProfileId = 0)
+    ).use { adminScenario ->
```

Comment:
remove unused variable
```suggestion
    ).use { 
```